signbank/dictionary/tests/test_adminviews.py

```diff
@@ -155,4 +159,51 @@ def test_delete(self):
         """Tests that DELETE doesn't work on search page."""
         response = self.client.delete(reverse('dictionary:admin_gloss_list'))
         # 405 Method Not Allowed
-        self.assertTrue(response.status_code == 405)
\ No newline at end of file
+        self.assertTrue(response.status_code == 405)
+
+
+class TestValidationResultsView(TestCase):
+    def setUp(self):
+        # Create user and add permissions
+        self.user = User.objects.create_user(username="test", email=None, password="test",
+                                             is_staff=True)
+        permission = Permission.objects.get(codename='search_gloss')
+        self.user.user_permissions.add(permission)
+
+        self.client = Client()
+        self.client.force_login(self.user)
+
+        # Create a gloss
+        # Migrations have id=1 already
+        self.signlanguage = SignLanguage.objects.create(pk=2, name="testsignlanguage",
+                                                        language_code_3char="tst")
+        self.dataset = Dataset.objects.create(name="testdataset", signlanguage=self.signlanguage)
+        self.gloss = Gloss.objects.create(idgloss="testgloss", dataset=self.dataset)
+
+        self.share_validation_aggregation_1 = ShareValidationAggregation.objects.create(
+            gloss=self.gloss, agrees=2, disagrees=5)
+        self.share_validation_aggregation_2 = ShareValidationAggregation.objects.create(
```

Comment:
**suggestion**: if the single and multiple results cases are being handled differently, they should both be covered in testing